src/tcgwars/logic/impl/gen4/DiamondPearl.groovy

```diff
@@ -973,15 +1172,18 @@ public enum DiamondPearl implements LogicCardInfo {
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              cantRetreat defending
             }
           }
           move "Ease Up", {
             text "40+ damage. Snorlax is now Asleep. If Snorlax is evolved from Munchlax, this attack does 40 damage plus 30 more damage."
             energyCost C, C, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 40
+              //TODO: Baby Evolution check (see Electivire/Roserade)
```

Comment:
Leaving a comment here this doesn't get lost from the previous related ones as well.